### DISCUSSION TITLE
Connect only one type of Wallet - Flutter

### DISCUSSION THREAD

**Discussion by MrCube72** [WalletConnect/walletconnect-monorepo#3892] [Developer Support]
Hi!

I'm trying to enable connect only MetaMask Wallet in my dApp, but the new version of WalletModal doesn't support this option. It seems "includedWalletIds" include not only the selected Wallets (in my case:
```dart
final Set<String> includedWalletIds = {
  'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96', // MetaMask
};
```
"excludedWalletIds" seems work, but V3 doesn't have the option exclude ALL other Wallets (like in V2:
```dart
recommendedWalletIds: {
      'c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96', // MetaMask
    },
    excludedWalletState: ExcludedWalletState.all,
```
 Do you plan any similar options?
Thanks...

**Comment by glitch-txs** [comment]
hi, could you create an issue in flutter repository for this? as a feature request. Thanks!
